src/Raven.Studio/wwwroot/App/views/database/tasks/exportDatabase.html

```diff
@@ -172,14 +172,24 @@ <h4>Select collections:</h4>
                             <div data-bind="compose: 'database/tasks/smugglerDatabaseRecord.html'"></div>
                         </div>
                     </div>
-                    <div class="form-group">
-                        <label class="control-label">Command</label>
-                        <div class="input-group">
-                        <span class="input-group-btn">
-                            <button class="btn btn-default" data-bind="click: copyCommandToClipboard"><i class="icon-copy"></i> <span>Copy to clipboard</span></button>
-                        </span>
-                            <input onClick="this.select();" class="form-control" readonly data-bind="value: exportCommand" />
+                    <div class="flex-horizontal" data-bind="with: $root">
+                        <div class="btn-group">
+                            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                                <span data-bind="text: effectiveCommandLabel"></span>
+                                <span class="caret margin-left margin-left-xs"></span>
+                            </button>
+                            <ul class="dropdown-menu">
+                                <li><a href="#" data-bind="click: effectiveCommandType.bind(effectiveCommandType, 'PowerShell')"><span data-bind="text: getCommandTypeLabel('PowerShell')"></span></a></li>
```

Comment:
let's expose list of command types in model and iterate though them - code will be more bullet proof.

Reply:
done